runtime/compiler/net/CommunicationStream.hpp

```diff
@@ -86,7 +86,10 @@ class CommunicationStream
       _ssl = ssl;
       }
 
+   // The first read is to read the message in the number of bytes of the full buffer capacity.
```

Comment:
Maybe we should expand more these comments:
"Build a message sent by a remote party by reading from the socket as much as possible (up to internal buffer capacity)"

"Build a message sent by a remote party by first reading the message size and then reading the rest of the message"